lib/manager/gradle-wrapper/extract.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  extractPackageFile,
+};
+
+function extractPackageFile(fileContent) {
+  logger.debug('gradle-wrapper.extractPackageFile()');
+  const lines = fileContent.split('\n');
+
+  let lineNumber = 0;
+  for (const line of lines) {
+    const match = line.match(
+      /^distributionUrl=https\\:\/\/services.gradle.org\/distributions\/gradle-((\d|\.)+)-(bin|all)\.zip\s*$/
```

Comment:
Will there ever be more than one of these lines in a file?

Reply:
No, it it can only be one.